Cargo.toml

```
@@ -1442,6 +1442,15 @@ description = "Illustrates creating and updating a button"
 category = "UI (User Interface)"
 wasm = true
 
+[[example]]
+name = "fallthrough"
```

example name could be more explicit
```suggestion
name = "window_fallthrough"
```